client/packages/electron/src/electron.ts

```diff
@@ -7,12 +7,59 @@ import {
   frontEndHostUrl,
   isProtocol,
 } from '@openmsupply-client/common/src/hooks/useNativeClient';
+import HID from 'node-hid';
+
 const SERVICE_TYPE = 'omsupply';
 const PROTOCOL_KEY = 'protocol';
 const CLIENT_VERSION_KEY = 'client_version';
 const HARDWARE_ID_KEY = 'hardware_id';
+const SUPPORTED_SCANNERS = [
+  {
+    vendorId: 1504,
+    vendorName: 'Zebra',
+    products: [{ id: 2194, model: 'DS2208' }],
+  },
+];
+
+class BarcodeScanner {
+  device: HID.HID | undefined;
+
+  constructor() {
+    this.findDevice();
+  }
+
+  findDevice() {
```

Comment:
Nit: to solve the potentially undefined device within this class, you could make this method the "constructor" and make the real constructor private:

```typescript
static findDevice(): BarcodeScanner | undefined 
```